### DISCUSSION TITLE
Is the provided Helm chart for testing purposes only?

### DISCUSSION THREAD

**Discussion by aalldridge** [eclipse-edc/Connector#1794] [Q&A] [answered]
Hello EDC contributors,

we are deploying a customised version of the EDC in k8s as a central component at our project [EuroDaT](https://gitlab.com/eurodat.org/trustee-platform). So far, we have been using custom Helm charts to so. Since you have since made your Helm chart more generic, we could probably import it as a dependency. Before we do so, I would like to know: Is this Helm chart for testing purposes only, or do you consider it a supported mode of installation? 

Cheers, Alexander

**Comment by jimmarino** [comment, marked as answer]
Hi,

Due to the highly customizable nature of the EDC, the only official distributions are the modules published to Maven central. Eventually, we will provide BOM modules. Downstream projects may choose to create containers, Helm charts, etc.
